eox_core/edxapp_wrapper/backends/users_h_v1.py

```diff
@@ -4,19 +4,28 @@
 Backend for the create_edxapp_user that works under the open-release/hawthorn.beta1 tag
 """
 from __future__ import absolute_import, unicode_literals
+
 import logging
 
-from django.db import transaction
-from django.contrib.auth import get_user_model
+from contentstore.views.user import _course_team_user
 from django.conf import settings
-from rest_framework.exceptions import APIException
-from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY  # pylint: disable=import-error
-from openedx.core.djangoapps.user_api.accounts.api import check_account_exists  # pylint: disable=import-error
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from openedx.core.djangoapps.lang_pref import \
```

Comment:
I would recommend to import in two lines like this:
```python
from openedx.core.djangoapps.lang_pref import (
    Mymodule
)
```

Applies for all of them